src/harberger-lib/components/Info.re

```diff
@@ -240,3 +241,168 @@ let make = (~tokenId: TokenId.t) => {
+    let displayName = UserProvider.useDisplayName(currentPatron);
+    let displayNameStr = UserProvider.displayNameToString(displayName);
+
+    let tokenName = tokenId->QlHooks.useWildcardName |||| "loading name";
+
+    let clearAndPush = RootProvider.useClearNonUrlStateAndPushRoute();
+
+    let currentUsdEthPrice = UsdPriceProvider.useUsdPrice();
+    let (totalPatronage, totalPatronageUsd) =
+      QlHooks.useAmountRaisedToken(tokenId)
+      ->mapd(("Loading", "Loading"), a =>
+          (
+            (a->Eth.get(Eth.Eth(`ether))->Float.fromString |||| 0.0)
+            ->toFixedWithPrecisionNoTrailingZeros(~digits=9),
+            currentUsdEthPrice->mapd("Loading", usdEthRate =>
+              a->Eth.get(Eth.Usd(usdEthRate, 2))
+            ),
+          )
+        );
+
+    let ratio = QlHooks.usePledgeRate(tokenId);
+
+    let monthlyRate = Js.Float.toString(ratio *. 100.);
+
+    <React.Fragment>
+      <div>
+        {if (ratio == 0.) {
+           <p>
+             "The monthly pledge rate will be revealed at launch."->restr
+           </p>;
+         } else {
+           <>
+             <small>
+               <strong>
+                 "Monthly Pledge Rate:"->restr
+                 <Rimble.Tooltip
+                   message={
+                     "This is the monthly percentage contribution of "
+                     ++ tokenName
+                     ++ "'s sale price that will go towards conservation of at risk animals. This is deducted continuously from the deposit and paid by the owner of the animal"
+                   }
+                   placement="top">
+                   <span> {js|ⓘ|js}->restr </span>
+                 </Rimble.Tooltip>
+               </strong>
+             </small>
+             <br />
+             {(monthlyRate ++ " %")->restr}
+           </>;
+         }}
+      </div>
+      {abandoned
+         ? <p>
+             <strong>
+               "Was abandoned by "->restr
+               <a
+                 onClick={e => {
+                   ReactEvent.Mouse.preventDefault(e);
+                   clearAndPush({j|/#user/$currentPatron|j});
+                 }}>
+                 displayNameStr->restr
+               </a>
+               <Rimble.Tooltip
+                 message={j|This happens when the user's deposit runs out for the wildcard.|j}
+                 placement="top">
+                 <span> {js|ⓘ|js}->restr </span>
+               </Rimble.Tooltip>
+             </strong>
+             <br />
+           </p>
+         : <p>
+             "This wildcard has never had a guardian - you can be the first."
+             ->React.string
+           </p>}
+      <p>
+        <small>
+          <strong>
+            {{
+               tokenName ++ "'s Patronage: ";
+             }
+             ->restr}
+            <Rimble.Tooltip
+              message={
+                "This is the total contribution that has been raised thanks to the wildcard, "
```

Comment:
thanks to `animal name`. Just an opinion so if you feel otherwise don't feel obliged to change